scripts/runcode.jl

```diff
@@ -329,17 +344,24 @@ function chamber(
     log_vfr_vector::Union{Float64,Vector{Float64}},
     depth_vector::Union{Float64,Vector{Float64}},
     output_dirname::String=get_timestamp(),
+    ;
+    plotfig::Bool=true,
 )::String
     path0 = joinpath(pwd(), output_dirname)
     mkdir(path0)
     to = TimerOutput()
     @timeit to "chamber" begin
-        for log_volume_km3 in log_volume_km3_vector,
-            InitialConc_H2O in InitialConc_H2O_vector,
-            InitialConc_CO2 in InitialConc_CO2_vector,
-            log_vfr in log_vfr_vector,
-            depth in depth_vector
-
+        Threads.@threads for (
+            log_volume_km3, InitialConc_H2O, InitialConc_CO2, log_vfr, depth
+        ) in collect(
+            Iterators.product(
+                log_volume_km3_vector,
+                InitialConc_H2O_vector,
+                InitialConc_CO2_vector,
+                log_vfr_vector,
+                depth_vector,
+            ),
+        )
```

Comment:
Added parallelization for array chamber.

Reply:
Was this tested locally?

Reply:
The thing is that I am not sure if the CI runs Julia in  a multi-core machine.

Reply:
> Was this tested locally?

Yes, it could run on my notebook. And if the number of Julia thread = 1, parallelization will not be activated.